### DISCUSSION TITLE
Languages "issue"

### DISCUSSION THREAD

**Discussion by z0grim** [beetbox/beets#4414] [Q&A] [answered]
Hi All!

https://musicbrainz.org/release/ed095f95-99e9-4eeb-9e5f-2ec39bd76a0d

I can't seem to track down how or why the Album name remains with Japanese characters. Initially both Artist or Album name was in Japanese, so I've started to edit the MB entry. Added localisation to both Artist and Album, but there is also an Album Group so I added it there as well, but it still keeps fetching the "Japanese version" of the Album name.

I'm not sure what I'm missing here.
`import:
    move: yes
    languages: en hu
`

`beet import -t ...`

`Tagging:
    Kiyoshi Yoshida - カイバ ORIGINAL SOUNDTRACK
URL:
    https://musicbrainz.org/release/ed095f95-99e9-4eeb-9e5f-2ec39bd76a0d
(Similarity: 100.0%) (CD, 2008, JP, vap, VPCY-13083C)`

**Comment by sampsyo** [comment, marked as answer]
The `languages` configuration only affects artist aliases. This artist does have an English alias:
https://musicbrainz.org/artist/ad46bbda-c1a4-4e2e-8459-b34a1a441a9b/aliases

…so that would explain how you're getting a transliterated artist name. We don't currently have a way of localizing release names (and in fact, I don't think I was aware that MusicBrainz supported localized release names).

**Reply by z0grim** [reply]
Thanks for the clarification, I assumed that alias worked on all levels not just Artist name. No wonder it did not work then.
